src/Hamiltonians/TwoBodyCorrelation.jl

```diff
@@ -0,0 +1,70 @@
+# Methods that need to be implemented:
+#   •  starting_address(::AbstractHamiltonian) - not needed
+"""
+    TwoBodyCorrelation(d::Int)
```

Comment:
```suggestion
    G2Correlator(d::Int)
```
I would suggest this name (or similar) because there are many other versions of two-body correlation functions that we may want to implement later.

Reply:
Sounds good. Function name changed.